docs/src/main/sphinx/connector/jdbc-authentication.fragment

```diff
@@ -0,0 +1,55 @@
+Data source authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector can provide credentials for the data source connection
+in multiple ways:
```

Comment:
Maybe say ... in the following ways:

Imo, "following" anticipates a list and explanation more than simply saying "multiple", which sounds like a stop.

Reply:
I like that, thanks!